opus/application/test_api/test_cart_api.py

```diff
@@ -778,7 +802,22 @@ def test__api_cart_remove_one_download(self):
         expected = {'recycled_count': 0, 'count': 1, 'error': False, 'reqno': 456}
         self._run_json_equal(url, expected)
         url = '/__cart/remove.json?opusid=co-iss-n1460961026&reqno=123456&download=1'
-        expected = {'recycled_count': 0, 'count': 0, 'error': False, 'reqno': 123456, "total_download_count": 0, "total_download_size": 0, "total_download_size_pretty": "0B", "product_cat_list": []}
+        expected = {'recycled_count': 0, 'count': 0, 'error': False, 'reqno': 123456, "total_download_count": 0, "total_download_size": 0, "total_download_size_pretty": "0B", "product_cat_dict": {}}
+        self._run_json_equal(url, expected)
+        url = '/__cart/status.json?reqno=789'
+        expected = {'recycled_count': 0, 'count': 0, 'reqno': 789}
+        self._run_json_equal(url, expected)
+
+    def test__api_cart_remove_download_ver(self):
```

Comment:
This is sort of a boring test, since the cart is empty after the remove, so the list of product types doesn't matter. Is there another case you can do where something is left in the cart after the remove?